src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackCompression.cs

```diff
@@ -25,5 +25,33 @@ public enum MessagePackCompression
         /// See also ThisLibraryExtensionTypeCodes.LZ4
         /// </devremarks>
         Lz4Block,
+
+        /// <summary>
+        /// Compresses an entire msgpack sequence as a multi contiguous block.
+        /// This is the fastest and results in the best LZ4 compression,
+        /// at the cost of copying the entire sequence when necessary to get contiguous memory.
```

Comment:
You evidently copied these two lines from the enum value defined above. But the two are mutually exclusive: they can't both the be fastest and best compression.

Reply:
Yes.
Actually, depending on the situation, which one is considered fastest is different, so I think it is better to delete the sentence in both.